README.md

```diff
@@ -184,20 +184,25 @@ Submits a map of custom `(key, value)` pairs to be included in the data for the
 Method accepts a single parameter, a JavaScript object with values of type `String`.
 `key` is limited to 80 characters and must be unique; `value` is limited to 1000 characters.
 
-#### areCookiesEnabled() => boolean
-Utility method to determine if cookies are enabled on the client's browser.  If cookies are not enabled, the
-quiq-chat library will not function properly.
+#### isStorageEnabled = () => boolean
+Utility function to tell the client if quiq-chat has the capability to set its required data.
+Note this will return true if quiq-chat is only able to set its data in a non-peristent way.
```

Comment:
Oh wait, no, sorry. I have it backwards.

isPersistentStorageEnabled > isStorageEnabled

isPersistentStorageEnabled will be true if something like cookies or localStorage are enabled
isStorageEnabled will be true only if it is able to set storage to some fallback way like `userDataStorage`